### PR TITLE
Feature EM-337: Button group of refresh and New line

### PR DESCRIPTION
**Pull request by Sankku**
Button group of refresh and New line that change depending of the view

Table View
image

Form View
image

### CODE REVIEW

File: packages/ComponentLibrary/src/components/Table/ExpandMenu/index.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { Menu, MenuItem, TextField } from '@mui/material';
+
+interface ExpandMenuProps {
+  anchorEl: null | HTMLElement;
+  onClose: () => void;
+  open: boolean;
+}
+
+const ExpandMenu: React.FC<ExpandMenuProps> = ({ anchorEl, onClose, open }) => {
```
Comment (LuuchoRocha) [inline packages/ComponentLibrary/src/components/Table/ExpandMenu/index.tsx:10]:
```suggestion
const ExpandMenu = ({ anchorEl, onClose, open }: ExpandMenuProps) => {
```

File: packages/ComponentLibrary/src/components/Table/ExpandMenu/index.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { Menu, MenuItem, TextField } from '@mui/material';
+
+interface ExpandMenuProps {
+  anchorEl: null | HTMLElement;
+  onClose: () => void;
+  open: boolean;
+}
+
```
Comment (LuuchoRocha) [inline packages/ComponentLibrary/src/components/Table/ExpandMenu/index.tsx:9]:
```suggestion
const bottomRightOrigin = {
  vertical: 'bottom',
  horizontal: 'right',
}
```

File: packages/ComponentLibrary/src/components/Table/ExpandMenu/index.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      }}
+      transformOrigin={{
+        vertical: 'top',
+        horizontal: 'left',
+      }}>
+      <MenuItem>
+        <TextField
+          value={inputValue}
+          onChange={handleInputChange}
+          placeholder="Enter value"
```
Comment (LuuchoRocha) [inline packages/ComponentLibrary/src/components/Table/ExpandMenu/index.tsx:34]: Please use the internationalization method `t('...')`

**Review by LuuchoRocha — CHANGES_REQUESTED** [review]

File: packages/MainUI/src/index.css [hunk 4]
```diff
@@ -18,7 +19,7 @@ html, body {
 #root {
   height: 100%;
   width: 100%;
-  background: linear-gradient(180deg, #C6D3FF 30%, #F5F6FA 100%);
```
Comment (tomirodeghiero) [inline packages/MainUI/src/index.css:21]: Move colors as contants

File: packages/storybook/src/stories/Components/Table/toolbarFormviewMock.tsx
Comment (tomirodeghiero) [inline packages/storybook/src/stories/Components/Table/toolbarFormviewMock.tsx:1]: Set the width and height to 16 as constant

File: packages/storybook/src/stories/Components/Table/toolbarMock.tsx
Comment (tomirodeghiero) [inline packages/storybook/src/stories/Components/Table/toolbarMock.tsx:1]: Set the width and height to 16 as constant

**Review by tomirodeghiero — CHANGES_REQUESTED** [review]

**Review by tomirodeghiero — APPROVED** [review]

**Review by leandroallemandismf — APPROVED** [review]

**Review by LuuchoRocha — APPROVED** [review]

**Pull request merged**